security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -15,6 +15,29 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
 
 To access the latest release, see xref:../../security/compliance_operator/compliance-operator-updating.adoc#olm-preparing-upgrade_compliance-operator-updating[Updating the Compliance Operator].
 
+[id="compliance-operator-release-notes-1-0-0"]
+== OpenShift Compliance Operator 1.0.0
+
+The following advisory is available for the OpenShift Compliance Operator 1.0.0:
+
+* link:https://access.redhat.com/errata/RHBA-2023:1682[RHBA-2023:1682 - OpenShift Compliance Operator bug fix update]
+
+[id="compliance-operator-1-0-0-new-features-and-enhancements"]
+=== New features and enhancements
+
+* The Compliance Operator is now stable and the release channel is upgraded to `stable`. Future releases will follow link:https://semver.org/[Semantic Versioning]. To access the latest release, see xref:../../security/compliance_operator/compliance-operator-updating.adoc#olm-preparing-upgrade_file-integrity-operator-updating[Updating the File Integrity Operator].
+
+[id="compliance-operator-1-0-0-bug-fixes"]
+=== Bug fixes
+
+* Before this update, the `rhcos4-ensure-logrotate-activated` rule failed after auto-remediation was applied in Red Hat Enterprise Linux (RHEL) 9 environments. With this update, rules with auto-remediation available have a `PASS` result after the auto-remedations are applied. (link:https://issues.redhat.com/browse/OCPBUGS-10769[*OCPBUGS-10769*])
```

Comment:
This sounds like the bug was a RHEL-9 BYO-OS issue which I don't think was the case, I think the issue would manifest also with RHEL-9 RHCOS. All in all, I think a simpler way to document this would be to say that this bug would affect 4.13 and newer clusters.

Reply:
I may just comment this out for now since 4.13 has not been released. After 4.13 release, we can uncomment it for the 4.13 only release.

Reply:
yes, I agree